lib/model.js

```diff
@@ -286,6 +286,9 @@ const _create = (data, definition) => {
   };
 
   definition.methods.saveWithId = async function(id, options) {
+    if (!this.isNew()) {
+      throw errors.Validation(new Error('saveWithId must receive a newly created object'));
+    }
```

Comment:
unsure if this should be a validation error or jus a generic `Error`

Reply:
just `new Error` - because this would be a error on the part of the person writing the code, not a user input error

Reply:
👍